utility/nsfs_server_utils.py

```diff
@@ -0,0 +1,152 @@
+"""
+Utility functions to run on the remote machine that hosts the NSFS server
+
+"""
+import json
+import logging
+import tempfile
+
+from common_ci_utils.templating import Templating
+from framework.ssh_connection_manager import SSHConnectionManager
+from noobaa_sa import constants
+from framework import config
+from noobaa_sa.exceptions import MissingFileOrDirectoryException
+
+log = logging.getLogger(__name__)
+
+
+def run_systemctl_command_on_nsfs_service(cmd):
```

Comment:
add doc string

Reply:
done in https://github.com/red-hat-storage/noobaa-sa-ci/pull/11/commits/bdf79c5c8a8f0626f87f6e83694334e53086fdbf